security/interceptor/security-interceptor-anonymous/src/main/java/org/codice/ddf/security/interceptor/AnonymousInterceptor.java

```diff
@@ -293,8 +297,21 @@ public void handleMessage(SoapMessage message) throws Fault {
                                     requireClientCert = evaluateExpression(xml, xpath);
 
                                 } else if (qName.equals(SP12Constants.SIGNED_SUPPORTING_TOKENS)) {
-                                    String xpath = "/SignedSupportingTokens/Policy/IssuedToken/RequestSecurityTokenTemplate/TokenType";
+                                    String xpath = "/SignedSupportingTokens/Policy/ExactlyOne/All/IssuedToken/RequestSecurityTokenTemplate/TokenType";
```

Comment:
You should be able to modify this xpath with a wild card to work for both cases.